src/platform/packages/shared/response-ops/alerts-table/components/default_cell_value.tsx

```diff
@@ -56,6 +60,7 @@ export const DefaultCellValue = ({
           href={http.basePath.prepend(
             `/app/management/insightsAndAlerting/triggersActions/rule/${ruleUuid}`
           )}
+          target={openLinksInNewTab ? '_blank' : undefined}
```

Comment:
Why do we need to open the links in a new tab?

Reply:
This was suggested as temporary replacement for the rule detail flyout with Joana, see https://github.com/elastic/kibana/pull/216076#issuecomment-2827907463

Reply:
Ah yeah. Thanks!